packages/compiler/core/nodeHost.browser.ts

```diff
@@ -0,0 +1,5 @@
+export const NodeHost = undefined;
```

Comment:
Wouldn't the tree shaking not load the NodeHost if you don't use it?

Reply:
That's not the behavior I was seeing unfortunately...

Reply:
I think the issue is primarily with vite dev, which I don't think does any optimizations. It's probable that this would not be needed without vite dev, but it's very nice to be able to make changes to the compiler and have them light up in the playground immediately without needing to do the production build (and I suspect will be a very nice way to develop on the openapi emitter).